qlib/FreetdsSqlUtil.qm

```diff
@@ -62,6 +62,25 @@ module FreetdsSqlUtil {
 
     @note This module requires the <a href="https://github.com/qorelanguage/module-sybase">freetds</a> binary module for communication with MS SQL Server and Sybase databases
 
+    @section freetds_sql_operations SQL Operations with the FreeTDS Driver
+
+    %FreetdsSqlUtil supports the following additional @ref select_option_hash "select option" when working with MS SQL Server databases:
```

Comment:
I suggest a big fat warning here (from ms sql server docs):
Because the SQL Server query optimizer typically selects the best execution plan for a query, we recommend that <join_hint>, <query_hint>, and <table_hint> be used only as a last resort by experienced developers and database administrators.